specification/synapse/resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/sqlServer.json

```diff
@@ -0,0 +1,1702 @@
+          "type": "boolean"
+        },
+        "queueDelayMs": {
+          "format": "int32",
+          "description": "Specifies the amount of time in milliseconds that can elapse before audit actions are forced to be processed.\r\nThe default minimum value is 1000 (1 second). The maximum is 2,147,483,647.",
+          "type": "integer"
+        }
+      }
+    },
+    "ExtendedServerBlobAuditingPolicy": {
+      "description": "An extended server blob auditing policy.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v1/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/ExtendedServerBlobAuditingPolicyProperties",
+          "description": "Resource properties.",
+          "x-ms-client-flatten": true
+        }
+      }
+    },
+    "ExtendedServerBlobAuditingPolicyListResult": {
+      "description": "A list of server extended auditing settings.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "Array of results.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ExtendedServerBlobAuditingPolicy"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "description": "Link to retrieve next page of results.",
+          "type": "string",
+          "readOnly": true
+        }
+      }
+    },
+    "ServerSecurityAlertPolicyProperties": {
+      "description": "Properties of a security alert policy.",
+      "required": [
+        "state"
+      ],
+      "type": "object",
+      "properties": {
+        "state": {
+          "description": "Specifies the state of the policy, whether it is enabled or disabled or a policy has not been applied yet on the specific server",
+          "enum": [
+            "New",
+            "Enabled",
+            "Disabled"
+          ],
+          "type": "string",
+          "x-ms-enum": {
+            "name": "SecurityAlertPolicyState",
+            "modelAsString": false
+          }
+        },
+        "disabledAlerts": {
+          "description": "Specifies an array of alerts that are disabled. Allowed values are: Sql_Injection, Sql_Injection_Vulnerability, Access_Anomaly, Data_Exfiltration, Unsafe_Action",
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "emailAddresses": {
```

Comment:
SQL has been going through a lot of iteration on the auditing/security related configs.  For example, you can no longer specify email addresses here in newer api versions.  Can you get someone from the SQL RP team to sign off on your proxy implementation in this PR before ARM looks at it to ensure they are OK with how this is being implemented on your end?

Reply:
I have request SQL Team to review once. But from SQL Server documentation ( https://docs.microsoft.com/en-us/rest/api/sql/serversecurityalertpolicies/createorupdate ) emailAddress is a valid field and I am getting the same as a response for this API from SQL Server side.

Reply:
Approved for SecurityAlertPolicies. We are returning the emailAdresses field and emailAdmins fiels to the policy since we got complaints from customers. I put few comments in the pr.

Reply:
Approved for auditingSettings and extendedAuditingSettings